crates/assists/src/handlers/remove_unused_param.rs

```diff
@@ -73,7 +73,7 @@ fn process_usage(
     let source_file = ctx.sema.parse(usage.file_range.file_id);
     let call_expr: ast::CallExpr =
         find_node_at_range(source_file.syntax(), usage.file_range.range)?;
-    if call_expr.expr()?.syntax().text_range() != usage.file_range.range {
+    if !call_expr.expr()?.syntax().text_range().contains_range(usage.file_range.range) {
```

Comment:
Let's extract lhs into a variable. Conditions crate induce congnitive load, so it makes sense to somewhat more aggressively simplify them.

LGTM otherwise (no more d+ needed)